python/ray/serve/api.py

```diff
@@ -401,6 +401,18 @@ class MyDeployment:
         logging_config=logging_config,
     )
     deployment_config.user_configured_option_names = set(user_configured_option_names)
+    if (
+        deployment_config.autoscaling_config
+        and "target_num_ongoing_requests_per_replica"
+        in deployment_config.autoscaling_config.dict(exclude_unset=True)
```

Comment:
Oh, yea sorry this got cut off a bit when I was checking the code. Yes, I think we should check the `autoscaling_config` coming from arg, since there are a few processing happening in between.

Reply:
done!